sys/dev/drm/freebsd/drm_os_freebsd.c

```diff
@@ -566,7 +566,7 @@ drm_fstub_do_mmap(struct file *file, const struct file_operations *fops,
 		sglist_append_phys(sg, (vm_paddr_t)vmap->vm_pfn << PAGE_SHIFT,
 		    vmap->vm_len);
```

Comment:
Is it wrong here as well then?  I suspect that the idea is that `fops->vmmap()` is supposed to have modified `vmap->vm_pfn` and `vmap->vm_len`, so I would perhaps drop this commit.

Reply:
I can drop it, it was just something I noticed while working on other things.

Reply:
To be clear, I don't actually think the current code is broken as I think the intention is that `fops->vmmap()` will populate both `vm_pfn` and `vm_len` and that a driver might even pick a different size?

Reply:
That's the intention; the bug I tried to fix (badly) was really that `fops->mmap()` doesn't actually populate `vm_len`. But until we have a consumer which actually triggers execution of that code, it should just be left alone.